src/main/java/calculator/model/inputValue/InputValue.java

```diff
@@ -0,0 +1,76 @@
+package calculator.model.inputValue;
+
+import java.util.ArrayList;
+
+public class InputValue {
+    ArrayList<Integer> positiveIntegers;
+    String delimiter = ",|:";
+    String customDelimiter;
+
+    String uncheckedInput;

```

Comment:
이 부분에 접근제한자와 `final` 키워드를 넣는건 어떨까요?
```suggestion
    private final ArrayList<Integer> positiveIntegers;
    private final String delimiter = ",|:";
    private final String customDelimiter;

    private final String uncheckedInput;
```